JLio.Core/Extensions/JsonMethods.cs

```diff
@@ -70,5 +70,36 @@ public static bool IsPropertyOfTypeArray(string propertyName, JObject o)
                 currentObject = (JObject) currentObject?[pathElement.ElementName];
             }
         }
+
+        public static bool RemoveItemFromTarget(JToken selectedValue)
+        {
+            var parent = selectedValue.Parent;
+            switch (parent?.Type)
+            {
+                case JTokenType.Property:
+                    parent.Remove();
```

Comment:
the selected value is the result of the JSONPath notation. So, when $.myObjecct.MyProperty is given to remove the selected value is the value of that property. not the property itself. the path indicated wich property to remove therefor the parent of the selected value should be removed. Removing of the value is not done like this, it can be done with the set command setting the property to JNull